src/proc/y12i-to-y16y16.cpp

```diff
@@ -15,7 +15,7 @@ namespace librealsense
     {
         auto count = width * height;
 #ifdef RS2_USE_CUDA
-        rscuda::split_frame_y16_y16_from_y12i_cuda(dest, count, reinterpret_cast<const y12i_pixel *>(source));
+        rscuda::split_frame_y16_y16_from_y12i_cuda(dest, count, reinterpret_cast<const rscuda::y12i_pixel *>(source));
```

Comment:
Intead of:
rscuda::split_frame_y16_y16_from_y12i_cuda(dest, count, reinterpret_cast<const rscuda::y12i_pixel *>(source));
Please try:
rscuda::split_frame_y16_y16_from_y12i_cuda<rscuda::y12i_pixel *>(dest, count, reinterpret_cast<const rscuda::y12i_pixel *>(source));